Create Fact “ground-nesting-bees” (#76)
mn-pollinators/pollinator-facts: src/facts/bee-nesting-underground.md

---

> title: Bee Nesting - Underground

Reviewer: I think we should rename to "Ground Nesting Bees"

image:

>   alt: Ground Nesting Bees

Reviewer: The alt text should describe the image rather than just repeat the fact title.

>   alt: Ground Nesting Bees

Reviewer 2: ```suggestion
  alt: Sandy, undisturbed soil with burrows
```

  src: ../images/beenestingunderground.jpg

>   license: CC by 2.0

Reviewer: ```suggestion
  license: CC BY 2.0
```

  url: https://www.flickr.com/photos/minicooper93402/6828563216
  creator: devra
source:

>   name: Wild Bee Nests and Building Wild Bee Houses Nesting Habitat

Reviewer: ```suggestion
  name: Wild bee nests
```

  url: https://www.beelab.umn.edu/wild-bees/wild-bees-and-houses

> category: Habitat

Reviewer: ```suggestion
category: habitat
```

---

> 30-40% of bees find cavities to live in, such as hollow branches/stems or dead trees. Inside, they use mud to create chambers for their eggs. Leave fallen trees/branches around your yard to attract bees.

Reviewer: I think you copied the wrong text here.
```suggestion
60-70% of bees live underground. They like to dig tunnels in sandy soil with nothing growing in it. Attract bees using bare, undisturbed soil around your yard.
```
